proto/streamlit/proto/MultiSelect.proto

```diff
@@ -27,7 +27,8 @@ message MultiSelect {
   string form_id = 6;
   repeated int32 value = 7;
   bool set_value = 8;
-  bool disabled = 9;
-  LabelVisibilityMessage label_visibility = 10;
-  int32 max_selections = 11;
+  int32 max_selections = 9;
+  string placeholder = 10;
+  bool disabled = 11;
+  LabelVisibilityMessage label_visibility = 12;
```

Comment:
Same comment here as in #6540 -- we'll need to keep tag numbers the same for existing fields and add placeholder as tag 12.

Reply:
https://github.com/streamlit/streamlit/pull/6901/commits/d14d3757294766d9d7704c3a926d4c26966d0519
I understood that it is not necessary to match the order of the arguments. Thanks for telling me.